parser/parser.y

```diff
@@ -10325,6 +10379,7 @@ AuthString:
 RoleNameString:
 	stringLit
 |	identifier
+|	"RESOURCE"
```

Comment:
It's failed when `create role resource` in [test](https://do.pingcap.net/jenkins/blue/organizations/jenkins/pingcap%2Ftidb%2Fghpr_mysql_test/detail/ghpr_mysql_test/7197/pipeline),  I fix it in a strange way. cc @tiancaiamao, do you have any suggest?

Reply:
I think using other unreserved keywords like "create role ACTION" fail also.  The work-around is to make a string literal to as. create role "resource".  I think we need to mention it in our release notes.

Reply:
Use non-keyword?

Reply:
Oh, `resource` is non-reserved keyword in MySQL.. 
https://dev.mysql.com/doc/refman/8.0/en/keywords.html

Reply:
I fix the test in https://github.com/pingcap/tidb-test/pull/2036. cc @tiancaiamao  @BornChanger